clients/venice-client/src/main/java/com/linkedin/venice/fastclient/ClientConfig.java

```diff
@@ -367,6 +388,7 @@ public String getClusterDiscoveryD2Service() {
     private StoreMetadataFetchMode storeMetadataFetchMode = StoreMetadataFetchMode.DA_VINCI_CLIENT_BASED_METADATA;
     private D2Client d2Client;
     private String clusterDiscoveryD2Service;
+    private boolean batchGetDefaultsToStreamingBatchGet = true;
```

Comment:
Should we keep the default to `false` until we get some validation on performance in cert?

Reply:
sounds good. We should probably clean up the single get based batchGet after making sure the streaming batch get works. Code looks sort of messy now.